api/librdb-ext-api.h

```diff
@@ -113,6 +113,12 @@ typedef struct RdbxToRespConf {
      * RESTORE command. */
     int delKeyBeforeWrite;
 
+    /* funcLibReplaceIfExist - If function-library with the same name is already
+     * exist in the target redis, then replace it rather than return failure.
+     * (Implemented in RESP by adding `REPLACE` flag to the `FUNCTION LOAD`
+     * command) */
+    int funcLibReplaceIfExist;
```

Comment:
i suppose the default is off, and i didn't see it being initialized, i assume we're using memset of calloc.